README.md

```diff
@@ -23,7 +23,23 @@ Sample Scanner Implementation
 
 Development Requirements
 -------------
-On fedora you can install requirements by issuing
+On Fedora or RHEL you can install requirements by issuing
 
-    dnf install ruby-devel rubygem-ffi rubygem-rake rubygem-bundler openscap
+    yum install ruby-devel rubygem-ffi rubygem-rake rubygem-bundler openscap
```

Comment:
Did you try this command on rhel7?

I might be wrong, but I thought things like rubygem-ffi are not available for RHEL (unless you enable some arcane channels...).

In other words, can you please assert that you have tried the command on RHEL and it passed, before we patch it? Thanks!

Reply:
@isimluk I have tried, but probably with EPEL enabled. Anyway, updated it so there are no suspicious "package not found" messages :)